recipes/mantidworkbench/bld.bat

```diff
@@ -8,6 +8,10 @@ cmake ^
     -DCONDA_ENV=True ^
     -DENABLE_PRECOMMIT=OFF ^
     -DENABLE_DOCS=ON ^
+    -DDOCS_DOTDIAGRAMS=ON ^
+    -DDOCS_SCREENSHOTS=ON ^

```

Comment:
`-DDOCS_SCREENSHOTS=ON ^` is duplicated below

Reply:
Ah yes! We must have realised that at some point.